frontend/src/modules/Shares/views/ShareView.js

```diff
@@ -65,6 +65,7 @@ import {
   UpdateRejectReason,
   UpdateRequestReason
 } from '../components';
+import { generateShareItemLabel } from '../../../utils/share';
```

Comment:
Relative imports should be replaced by global imports

Reply:
Done!
Could you also tell me why this is best practice? Just curious!

Reply:
So it means that you can import directly from ../utils, like this:
`import { generateShareItemLabel } from '../utils';`

Reply:
Absolute imports make the hell of ../../../../../ way easier (check this [post](https://medium.com/geekculture/making-life-easier-with-absolute-imports-react-in-javascript-and-typescript-bbdab8a8a3a1)).

In my short experience using them, it makes it easier to move around files knowing that no matter where they are placed the imports do not change. After 10 mins you already know your "packages" by heart and can reference them very fast. In the frontend for example, we always import components from `design`, `services`, `utils`. 

And as a positive side effect, it makes you reflect on where you place the code. We still use relative imports inside each of the modules, because it is code that is used only inside the module. Globally exposed code is meant to be re-used everywhere. This differentiation kind of makes it evident when module specific code is made global

Reply:
Thanks for the explanation! 
That was very helpful.